packit.yaml

```diff
@@ -7,9 +7,9 @@ actions:
     - cp tools/cockpit.spec .
 
   create-archive:
-    # The sandcastle doesn't have enough ram to run webpack, so wait
-    # until the webpack-jumpstart workflow has run and grab the result.
-    - tools/webpack-jumpstart --wait --rebase
+    # The sandcastle doesn't have enough ram to run esbuild, so wait
+    # until the build-jumpstart workflow has run and grab the result.
+    - tools/build-jumpstart --wait --rebase
```

Comment:
You said with esbuild, building all webpacks just takes a few seconds.. We can then just drop this. The sandcastle container is no more, and RAM limits are not an issue. I suppose we can drop the whole jumpstart machinery completely, but fine to do this as a follow-up.

Reply:
Right, I thought followup is better as this jumpstart script is not affected at all by the build changes